lib/java/com/google/android/material/timepicker/res/layout/material_textinput_timepicker.xml

```diff
@@ -16,17 +16,10 @@
 -->
 
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
   android:layout_width="wrap_content"
   android:layout_height="wrap_content"
   android:baselineAligned="false"
-  android:layoutDirection="ltr"
```

Comment:
Here is `material_textinput_timepicker` before the fix:
```
layout:
-------
material_textinput_timepicker - Horizontal LinearLayout - ltr
├─ material_timepicker_textinput_display - ltr (inherited)
└─ material_clock_period_toggle - ltr (inherited)

layout-ldrtl:
-------------
material_textinput_timepicker - Horizontal LinearLayout - ltr
├─ material_clock_period_toggle - ltr (inherited)
└─ material_timepicker_textinput_display - ltr (inherited)
```

And `material_textinput_timepicker` after the fix:
```
layout:
-------
material_textinput_timepicker - Horizontal LinearLayout - ltr/rtl
├─ material_timepicker_textinput_display - ltr
└─ material_clock_period_toggle - ltr/rtl (inherited)
```

The only difference is that for RTL, `material_clock_period_toggle` before the fix has layoutDirection=ltr (incorrect, by the way), and after the fix - layoutDirection=rtl (correct). It's worth noting, though, it won't affect the TimePicker appearance, because `material_clock_period_toggle` looks the same in both LTR and RTL mode.

Reply:
Gotcha. Thank you so much for the explanation!